k8s/ietfweb/secrets.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secrets-env
+type: Opaque
+stringData:
+  IETFWWW_ADMINS: |-
```

Comment:
Maybe we should set this to `null` here. I have no strong opinion.

Reply:
I'll set it to null in both `ietfweb` and `iabweb`.